py3status/modules/window_title.py

```diff
@@ -51,7 +51,12 @@ def __init__(self):
     def window_title(self):
         window = find_focused(i3.get_tree())
 
+        if not window or window.get('name') is None or \
+           str(window.get('num')) == window.get('name'):
```

Comment:
` str(window.get('num')) == window.get('name')`

what is this testing for and does it change the output of this module from the existing one?